content/en/quality_gates/explorer/facets.md

```diff
@@ -0,0 +1,124 @@
+
+[Creating facets](#creating-facets) is **not required** for [searching quality gates][5]. Autocomplete capabilities use existing facets, but also any input that matches incoming quality gates applies.
+
+The [Quality Gates Explorer][4] includes out-of-the-box facets such as `Test Status`, `Test Service`, `CI Status`, and `CI Provider`. You can use facets in the Quality Gates Explorer to [search for and filter quality gates][5].
+
+{{< tabs >}}
+{{% tab "Gates" %}}
+
+Navigate to [**CI** > **Quality Gate Executions**][101] and select **Gates** to access the list of [Quality Gates facets][102].
+
+{{< img src="quality_gates/explorer/facets_gates.png" text="Facets list for gates on the Quality Gates Explorer" style="width:100%" >}}
+
+[101]: https://app.datadoghq.com/ci/quality-gates/executions
+[102]: /quality_gates/explorer/?tab=gates
+
+{{% /tab %}}
+{{% tab "Rules" %}}
+
+Navigate to [**CI** > **Quality Gate Executions**][101] and select **Rules** to access the list of [Quality Gates facets][102].
+
+{{< img src="quality_gates/explorer/facets_rules.png" text="Facets list for rules on the Quality Gates Explorer" style="width:100%" >}}
+
+[101]: https://app.datadoghq.com/ci/quality-gates/executions
+[102]: /quality_gates/explorer/?tab=rules
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Qualitative facets
+
+Use qualitative facets to:
+
+- **Get relative insights** for values.
+- **Count unique values**.
+- Frequently **filter** your quality gates against particular values. For example, create a facet on an environment tag to scope troubleshooting down to development, staging, or production environments.<br>
+
+**Note:** Although facets are not required for filtering on tags, defining facets for tags that you often use during investigations can help reduce your time to resolution.
+
+### Quantitative measures
+
+Use quantitative measures to:
+
+- **Aggregate** values from multiple quality gates.
+- **Range filter** your quality gates.
+- **Sort** your quality gates against a value.
+
+#### Types
+
+Measures support long integers and double values.
+
+#### Units
+
+Measures support units (**time** in seconds or **size** in bytes) for handling of orders of magnitude at query time and display time. The unit is a property of the measure itself, not of the field.
+
+For example, consider a `duration` measure in nanoseconds. Suppose test runs from `service:A` have `duration:10000000`, meaning `10 milliseconds`. Supposed test runs from `service:B` have `duration:5000000`, meaning `5 milliseconds`. Use `duration:>2ms` to consistently query test run tags from both services at once. For more information about search queries, see [Search Syntax][6].
```

Comment:
this example is from Testing - we can probably remove this "Units" section altogether, since we don't have the concept of adding measure tags to Quality Gate executions